src/main/java/uk/gov/companieshouse/api/strikeoffobjections/controller/ObjectionController.java

```diff
@@ -134,7 +136,8 @@ public ObjectionController(PluggableResponseEntityFactory responseEntityFactory,
             @PathVariable("companyNumber") String companyNumber,
             @RequestHeader(value = ERIC_REQUEST_ID) String requestId,
             @RequestHeader(value = ERIC_IDENTITY) String ericUserId,
-            @RequestHeader(value = ERIC_AUTHORISED_USER) String ericUserDetails
+            @RequestHeader(value = ERIC_AUTHORISED_USER) String ericUserDetails,
+            @RequestBody ObjectionCreate objectionCreate
```

Comment:
Very minor I know, but as this change will modify the service interface, it is probably a good a time as any to bump the Major version as any apps built against the old interface will no longer work as expected

Reply:
Will do.

Reply:
Changed version from 0.1 to 0.2